CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+ advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+ address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+ professional setting
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at chris@blueprintforfreespeech.net. All
```

Comment:
I'm going to comment on my own pull request. @z3ndrag0n is it possible to setup a ricochet address at blueprint that forwards all emails to all four of us? It would be better if it wasn't just going to me.

Reply:
Yep - there is already ricochet which goes to Suelette, Peter and Eleanor. I suggest we change that (and can keep Suelette and Eleanor on). I also suggest we have a seperate one ricochet-conduct or similar.

Reply:
I have updated ricochet and created ricochet-conduct.

Reply:
Update

> Please change [chris@blueprintforfreespeech.net](mailto:chris@blueprintforfreespeech.net) to [ricochet-conduct@blueprintforfreespeech.net](mailto:ricochet-conduct@blueprintforfreespeech.net). Also a suggestion, change "abuser" to "person responsible" in CONTRIBUTE.md. Not all conduct violating the code of conduct is abuse.

Done see commit 0f014c0